ament_copyright/ament_copyright/parser.py

```diff
@@ -166,7 +166,7 @@ def search_copyright_information(content):
     year = '\d{4}'
     year_range = '%s-%s' % (year, year)
     year_or_year_range = '(?:%s|%s)' % (year, year_range)
-    pattern = '^[^\n\r]?\s*Copyright\s+(%s(?:,\s*%s)*)\s+([^\n\r]+)$' % \
+    pattern = '^[^\n\r]?\s*Copyright\s+(%s(?:,\s*%s)*),?\s+([^\n\r]+)$' % \
```

Comment:
Can you give an example what wasn't accepted before and is accepted with this change.